### ISSUE TITLE
Add support for the HDF5 Image Specification (eg RGB data)

### ISSUE THREAD

**Issue by planetmarshall** [silx-kit/h5web#649]
It would be useful to display RGB data as a bitmap - ie identifying datasets with the `IMAGE` and `IMAGE_SUBCLASS` attributes.

See [HDF5 Image Sepcification](https://support.hdfgroup.org/HDF5/doc/ADGuide/ImageSpec.html)

**Comment by axelboc** [comment]
Hi @planetmarshall, thanks for brining this to our attention. The image specification seems quite complex, so I doubt we can support it fully, but we'd be willing to implement a very basic subset of it - e.g. to support RGB data as you mention.

Which features of the specification (attributes and exact values) are you interested in specifically? And do you have an example HDF5 file we could use for testing purposes?

**Comment by planetmarshall** [comment]
Yes - basic support for RGB data would be perfect. I'm not familiar with the code, but if I can see how support for that is introduced I can always contribute additional support myself.

An [example h5](https://github.com/silx-kit/h5web/files/6514829/image_test.h5.gz) file is attached, it is created from the [Mandrill](https://github.com/scijs/baboon-image) test image as follows:

```.py
import skimage.io
import h5py

rgb = skimage.io.imread("https://raw.githubusercontent.com/mikolalysenko/baboon-image/master/baboon.png")
with h5py.File("image_test.h5", "w") as fp:
    image = fp.create_dataset("mandrill", data=rgb)
    image.attrs["CLASS"] = "IMAGE"
    image.attrs["SUBCLASS"] = len("IMAGE_TRUECOLOR")
    image.attrs["IMAGE_VERSION"] = "1.2"
```

When viewed with, for example, the Jupyter viewer ideally I would be able to visualize the dataset as an RGB image. 

Additionally it would be helpful to be able to switch between RGB and BGR interpretation (as some image are inverted on that axis, for example images from OpenCV) - in a similar fashion to how colour maps can be selected, however again this is always something I can contribute myself.

Many thanks.

**Comment by vasole** [comment]
@t20100 

In principle the interpretation attribute can be used to interpret the RGB case. I do not remember if it is already supported in silx.

Should we check for the presence of the interpretation attribute **and** if not present the HDF5 Image attributes? I'm in favor.

**Comment by planetmarshall** [comment]
Another issue which I've only just noticed is that the y-axis should be inverted for image data, so that (0,0) is at the top left.

**Comment by loichuder** [comment]
@planetmarshall Support for RGB data landed in v0.0.24 of h5web and v0.0.8 of jupyterlab-h5web.

Could you have a look and tell us if that fits your needs ?

**Comment by axelboc** [comment]
Closing the issue, but feel free to re-open if something is missing.
